### PR TITLE
Add a DOI.

### PR DESCRIPTION
**Pull request by bangerth**
/rebuild

### CODE REVIEW

File: publications-2016.bib [hunk 1]
```diff
@@ -1723,6 +1723,7 @@ @InProceedings{Salmoiraghi2016
   booktitle = {Proceedings of the ECCOMAS Congress 2016, VII European Conference on Computational Methods in Applied Sciences and Engineering},
   year      = {2016},
   url       = {http://preprints.sissa.it/xmlui/handle/1963/35179},
+  doi       = {10.7712/100016.1867.8680}
```
Comment (masterleinad) [inline publications-2016.bib]: I guess we can add some more info. This is what google scholar says:
```
@inproceedings{salmoiraghi2016advances,
  title={Advances in geometrical parametrization and reduced order models and methods for computational fluid dynamics problems in applied sciences and engineering: overview and perspectives},
  author={Salmoiraghi, Filippo and Ballarin, Francesco and Corsi, Giovanni and Mola, Andrea and Tezzele, Marco and Rozza, Gianluigi and others},
  booktitle={VII European Congress on Computational Methods in Applied Sciences and Engineering},
  volume={1},
  pages={1013--1031},
  year={2016},
  organization={Institute of Structural Analysis and Antiseismic Research School of Civil Engineering National Technical University of Athens (NTUA) Greece}
}
```

**Comment by bangerth** [comment]
Added.

File: publications-2016.bib [hunk 2]
```diff
@@ -1721,8 +1721,12 @@ @InProceedings{Salmoiraghi2016
   author    = {Salmoiraghi, F and Ballarin, F and Corsi, G and Mola, A and Tezzele, M},
   title     = {Advances in geometrical parametrization and reduced order models and methods for computational fluid dynamics problems in applied sciences and engineering},
   booktitle = {Proceedings of the ECCOMAS Congress 2016, VII European Conference on Computational Methods in Applied Sciences and Engineering},
+ organization={Institute of Structural Analysis and Antiseismic Research School of Civil Engineering National Technical University of Athens (NTUA) Greece}
```
Comment (masterleinad) [inline publications-2016.bib]:
```suggestion
 organization={Institute of Structural Analysis and Antiseismic Research School of Civil Engineering National Technical University of Athens (NTUA) Greece},
```

**Comment by bangerth** [comment]
Fixed that as well :-(

**Pull request merged**